packages/world-postgres/src/drizzle/migrations/0002_add_expired_at.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "workflow"."workflow_runs" ADD COLUMN "expired_at" timestamp;
```

Comment:
we also need to update vercel and local world no? is it possible to include a test anywhere, ideally e2e/world testing, so we can enforce other worlds implement this correctly too?